src/test_utils/katana/mod.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(deprecated)]
```

Comment:
why deprecated?

Reply:
`dojo_test_utils::sequencer::SequencerConfig` is in the process of removal, we should open an issue about this, will open this right now

https://github.com/dojoengine/dojo/blob/60c1575927db63e98ababda56c5ffa9f6337fb56/crates/katana/core/src/sequencer.rs